docs/pages/guides/how-to-use-scss.md

```diff
@@ -0,0 +1,183 @@
+
+2. Open `.angular-cli.json` located at the project root and change the styles configuration to `"styles": [ "scss/app.scss"]`
+
+3. Verify that the `styleExt` is set to `scss`
+
+4. Create a folder named `scss` under the `src` folder and a also create the main SCSS file named `app.scss`
+
+The Basic SCSS configuration is now complete.
+
+<br>
+
+## Installing Fundamental UI via NPM
+
+Fundamental UI is currently available as an [NPM package](https://www.npmjs.com/package/fundamental-ui){:target="_blank"} and a compiled and minified [CDN file](https://unpkg.com/fundamental-ui@1.0.0-beta-2/dist/fundamental-ui.min.css){:target="_blank"}. The following steps will cover how to install it via NPM.
+
+1. Type `npm install fundamental-ui --save-dev`. This will install the package as a dev dependency.
+2. Verify the installation was successful by typing `npm list fundamental-ui`
+
+Fundamental UI installation is now complete
+
+<br>
+
+## Configuring and Importing SCSS source
+
+1. Open `scss/app.scss` file
+2. Add the following line of code to define the icons path: <br><br> `$fd-icons-path: "~fundamental-ui/scss/icons/";`
+3. Add the following link of code to import scss source file: <br><br> `@import "~fundamental-ui/scss/all.scss";`
+
+Importing Fundamental UI SCSS is now complete
+
+> **Note:** In order to render the icons correctly, you need to define the `$fd-icons-path` variable.
+
+At this point, the contents of your `scss/app.scss` should look like this:
+
+{% highlight css %}
+
+$fd-icons-path: "~fundamental-ui/scss/icons/";
+@import "~fundamental-ui/scss/all.scss";
+
+{% endhighlight %}
+
+You can continue to build out your SCSS workflow based on your project needs.
+
+<br>
+
+## Selective Imports
+Fundamental UI is divided into several functional pieces. You may choose to import key features selectively to reduce the SCSS overhead in your project.
+
+Here is an overview of the SCSS structure -
+
+{% highlight css %}
+scss/
+ |- components/
+ |- core/
+ |- helpers/
+ |- icons/
+ |- layout/
+ |- theme/
+ all.scss
+ components.scss
+ core.scss
+ helpers.scss
+ icons.scss
+ layout.scss
+{% endhighlight %}
+
+- `all.scss`  includes everything
+- `components.scss` SCSS source for each each components. The components are decoupled  with little or no dependency on other components.
+- `core.scss` Gives you core functionality such as access to settings, functions and mix-ins.
+- `helpers.scss` includes various helper functions
+- `icons.scss` Fundamental UI's custom icon library
+- `layout.scss` App level layout components such as top Overview of Core features bar, side nav, containers, panels, etc.
+
+You can choose to selectively import parts of the library that best fits the need of your project. Please note that components, icons, layout and helpers have a dependency on `core.scss`. Therefore, `core.scss` should always be included in case if you are not importing `all.scss`
+
```

Comment:
This isn't exactly right. The dependency is on settings not on core. It's just that `core` brings those along for free — actually something we should address.`*` All of the groups import settings, functions, etc. themselves. You should be able to use `layout` without `core`.

If they want to cherry-pick the most important thing is that they include the theme, otherwise they will not get the fonts.

```
$fd-icons-path: "~fundamental-ui/scss/icons/";
@import "~fundamental-ui/scss/theme/techne";

@import "~fundamental-ui/scss/core";
```

What's missing here too is how to use the SCSS in a component SCSS file. To be able to access all the variables, functions and mixins, the basic setup for a component SCSS — `app.component.scss` — is to explicility import those dependencies every time.

```
@import "~fundamental-ui/scss/core/settings";
@import "~fundamental-ui/scss/core/mixins";
@import "~fundamental-ui/scss/core/functions";
```

`*` The thing we do not want is to import `"~fundamental-ui/scss/core/core"` inside component SCSS. That will duplicate all of the elements and forms. This approach should be revisited. We could better expose the mixins and functions in a future release.